src/Hl7.Fhir.R5.Core/ElementModel/PocoBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿/* 
+ * Copyright (c) 2018, Firely (info@fire.ly) and contributors
```

Comment:
R5.Core contains quite a few files that are not present in the others (HumanName.cs, IConformaceResource.cs, Initializer.cs, Questionnaire.cs), I cannot quickly find out why this is?